ansible/modules/hashivault/hashivault_approle_role_secret.py

```diff
@@ -145,7 +145,9 @@ def hashivault_approle_role_secret_create(module):
             result = client.create_role_custom_secret_id(role_name=name,
                                                              mount_point=mount_point,
                                                              secret_id=custom_secret_id,
-                                                             meta=metadata)
+                                                             meta=metadata,
```

Comment:
I'm not seeing in hvac where it takes these extra parameters. Is this in a release after 0.10.0?